docs/joss/paper.md

```diff
@@ -0,0 +1,211 @@
+The spherical harmonic transform is grid-flexible. Any iso-latitude ring-based
+grid can be used and new grids can be externally defined and passed in
+as an argument. Many grids are already implemented: the conventional
+Gaussian grid, a regular longitude-latitude grid, 
+the octahedral Gaussian grid [@Malardel2016], the octahedral
+Clenshaw-Curtis grid [@Hotta2018], and the HEALPix grid [@Gorski2005].
+Both SpeedyWeather.jl and its spherical harmonic transform SpeedyTransforms are also
+number format-flexible. 32-bit single-precision floating-point numbers
+(Float32) are the default as adopted by other modelling efforts [@Vana2017; @Nakano2018],
+but Float64 and other custom number formats can be used with a single
+code basis [@Klower2022; @Klower2020].
+Julia will compile to the choice of the number format, the grid,
+and and other model components just-in-time. A simple parallelisation
+across vertical layers is supported with Julia's multi threading.
+Output is stored as NetCDF files using
+[NCDatasets.jl](https://github.com/Alexander-Barth/NCDatasets.jl).
+
+# Statement of need
+
+SpeedyWeather.jl is a fresh approach to atmospheric
+models that have been very influential in many areas of scientific 
+and high-performance computing as well as climate change mitigation and adaptation.
+Most weather, ocean and climate models are written in Fortran and have been developed over
+decades. From this tradition follows a specific programming style and
+associated user interface. Running a simulation in Fortran and analysing the
+data in Python makes it virtually impossible to interact with various model
+components interactively. Furthermore, data-driven climate modelling
+[@Rasp2018; @Schneider2023], which replaces existing model components with machine learning
+is difficult due to the lack of established deep learning frameworks in Fortran [@Innes2019].
+Let alone online learning, which trains a neural network-based component together
+with the rest of the model, accounting for interactions between components.
+Gradients, necessary to optimize training, can be computed 
+with automatic differentiation [@Moses2020], but only if differentiable functions
+in a coherent language framework are provided.
+
+We hope to provide with SpeedyWeather.jl a first test platform for data-driven
+atmospheric modelling and in general an interactive model that makes difficult
+problems easy to simulate. Climate models that are user-friendly, trainable,
+but also easily extensible will suddenly make many complex
+research ideas possible.
+
+![Relative vorticity simulated with the shallow water model in SpeedyWeather.jl.
+The simulation used a spectral resolution of T1023 (about 20 km) and Float32
+arithmetic on an octahedral Clenshaw-Curtis grid [@Hotta2018]. Relative vorticity
+is visualised with Matplotlib [@Hunter2007] and Cartopy [@Cartopy] using a
+transparent-to-white colormap to mimic the appearance of clouds. Underlain is
```

Comment:
```suggestion
transparent-to-white colormap to mimic the appearance of clouds. Underlaid is
```
I think you want the past particle of underlay rather than underlie, since you put the thing there.

Reply:
Haha, yes, English is difficult!